main/urls.py

```diff
@@ -12,6 +12,7 @@
     path('', IndexView, name='home'),
     path('doctor/', IndexViewDoctor, name='home_doctor'),
     path('receptionist/', IndexViewReceptionist, name='home_receptionist'),
+    path('receptionist/appointments/', Recep_Appointments, name='recep_appointments'),
```

Comment:
look at the naming scheme if the views above this line, you should follow the same convention of naming.